apps/bootloader.js

```diff
@@ -101,12 +101,28 @@ if (startapp) {
   function drawWidgets() {
     for (var w of WIDGETS) w.draw();
   }
-  var clockApps = require("Storage").list().filter(a=>a[0]=='+').map(app=>{
-    try { return require("Storage").readJSON(app); }
-    catch (e) {}
-  }).filter(app=>app.type=="clock").sort((a, b) => a.sortorder - b.sortorder);
-  if (clockApps && clockApps.length > 0) eval(require("Storage").read(clockApps[0].src));
+  var settings;
+  try {
+    settings = require("Storage").readJSON('@setting');
+  } catch (e) {
+    settings = {}
+  }
+  var clockApp = settings.clock;
+  if (clockApp) {
+      clockApp = require("Storage").read(clockApp)
+  }
+  if (!clockApp) {
+    var clockApps = require("Storage").list().filter(a=>a[0]=='+').map(app=>{
+      try { return require("Storage").readJSON(app); }
+      catch (e) {}
+    }).filter(app=>app.type=="clock").sort((a, b) => a.sortorder - b.sortorder);
```

Comment:
Same here with `===`,
and I see mixed `"` and `'` in this file, can we choose one of those, but that's minor, just for consistency :)

Reply:
Again - this code was from `firmware.js`. Happy to go for consistency, but that needs considering across all of the existing code.

Reply:
@ra100 I'm really not sure this is helpful. It just feels like you're suggesting changes for the sake of picking holes in it.

Reply:
@gfwilliams I'm sorry, I'll try to refrain from these not helpful comments. I guess, I can create PR if it "pains" me so much :)

Reply:
:) Yeah, I agree it's definitely good practice to use `===` regardless of the minor speed difference (which may change in the future anyway). I think there's a bit of a balance of code quality vs friendliness to make in each project, and with something like Bangle.js everybody is contributing for fun so I think it's important not to be too picky.

But also given Nick's background I have quite a lot of confidence in his code :) Checking up on stuff like 'what happens if the clock app is removed' is super helpful though - so thanks for going through this.